diff --git a/RaylibUI/Initialization/GameFileLocatorScreen.cs b/RaylibUI/Initialization/GameFileLocatorScreen.cs
--- a/RaylibUI/Initialization/GameFileLocatorScreen.cs
+++ b/RaylibUI/Initialization/GameFileLocatorScreen.cs
@@ -1,5 +1,6 @@
 
 using Civ2engine;
+using Raylib_CSharp.Colors;
 using Raylib_CSharp.Images;
 
 namespace RaylibUI;
@@ -11,7 +12,7 @@ public class GameFileLocatorScreen : BaseScreen
 {
     public GameFileLocatorScreen(Main host, Action onSelect, Action shutdownApp) : base(host)
     {
-        ImageUtils.InnerWallpaper = Image.Load(AssetPaths.Resolve("stripe.png"));
+        ImageUtils.InnerWallpaper = Image.GenColor(16, 16, new Color(232, 232, 232, 255));
         ImageUtils.OuterWallpaper = Image.Load(AssetPaths.Resolve("SteelGrey.png"));
         ShowDialog(new FileDialog(host,"Please select Civ 2 data folder", Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), Settings.IsValidRoot, (fileName) =>
         {
